.gitlab-ci.yml

```diff
@@ -139,13 +139,13 @@ variables:
   S3_DSD6_URI: s3://dsd6-staging
   RELEASE_VERSION_6: nightly
   RELEASE_VERSION_7: nightly-a7
-  DATADOG_AGENT_BUILDIMAGES: v13321514-644fed4
+  DATADOG_AGENT_BUILDIMAGES: v13609860-94ebcfb
```

Comment:
This still needs to be updated with tags from the main branch, will be done once the PR on build images side is merged

Reply:
Done in cb1f16e70e20711fe8a19fc80b8c648477eb1992